ChatGPT/legos/chatgpt_get_token_count/chatgpt_get_token_count.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+
+##
+##  Copyright (c) 2023 unSkript, Inc
+##  All rights reserved.
+##
+#Note - this requires the tiktoken library to be installed
+from pydantic import BaseModel, Field
+import tiktoken
```

Comment:
Did we add this library to the requirements file in unskript?  Any new package needs to be added there.